.github/workflows/smoke_test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
Can't we write an actual integration tests with ScalaTest instead https://www.playframework.com/documentation/2.8.x/ScalaTestingYourApplication ?

There you can also start a real webserver and execute those tests

Reply:
yeah got your points, I think this is fine for now.

If we decide to add more tests we could maybe run this test only if the env var is set for the access token?